sources/packages/backend/libs/integrations/src/cas/models/cas-service.model.ts

```diff
@@ -47,6 +49,32 @@ export class CASAuthDetails {
   expires_in: number;
 }
 
+/**
+ * Default amount of time to a token be expired and be
+ * considered candidate to be renewed.
+ */
+const CAS_TOKEN_RENEWAL_SECONDS = 60;
```

Comment:
in system configuration constants ?

Reply:
I do not see a value in moving this. It is a private const only used to create some difference for the actual expiration time.